### PR TITLE
docs: update tss-library proposal with new modules and packages names

### PR DESCRIPTION
**Pull request by mxtartaglia-sl**
Finishes: #14390

### CODE REVIEW

**Review by edward-swirldslabs — APPROVED** [review]

File: platform-sdk/docs/proposals/TSS-Library/TSS-Library.md [hunk 1]
```diff
@@ -268,34 +276,47 @@ Given the input:
...
 10. `inner` = 0
 11. For each pair (`i`:index ; `k`:commitment value)  in `statement.polynomial_commitment`:
-    
+
     11.1.  `term` = For each `sid` in `statements.sids`: sum (`sid` raised to the power of `i`) multiplied by (`x` raised to the power of `sid`)
 
     11.2.  `inner` += `k` multiplied by `term`
+
 12. `lhs` = (`inner` multiplied by `x'`) add `proof.A`
+
```
Comment (jsync-swirlds) [inline platform-sdk/docs/proposals/TSS-Library/TSS-Library.md:285]: Did we intend to break the ordered list here?  These extra lines make the spacing of the list inconsistent and a bit stutter-ish.

File: platform-sdk/docs/proposals/TSS-Library/TSS-Library.md [hunk 2]
```diff
@@ -223,29 +232,28 @@ Given the input:
...
+Return the proof composed of `F`, `A`, `Y`, `z_r`, and `z_a`
 
 ##### Outside of scope
+
 Using an established channel, each participant will broadcast a single message to be received by all participants
 while waiting to receive other participants' messages. This functionality is critical for the protocol to work but needs to be handled outside the library.
 Each participant will validate the received message against the commitment and the NIZKs proof. Invalid messages need to be discarded.
 
 ##### Validation of TssMessage
+
```
Comment (jsync-swirlds) [inline platform-sdk/docs/proposals/TSS-Library/TSS-Library.md:256]: Is the addition of all these extra lines after headings intentional, or did Spotless do that?
(If spotless did this, then we need to fix spotless; this just looks terrible and is opposite to what every file we have does currently)

**Review by jsync-swirlds — COMMENTED** [review]
The formatting looks a bit broken.  Is that spotless?

**Review by anthony-swirldslabs — APPROVED** [review]
+1

**Comment by mxtartaglia-sl** [comment]
@jsync-swirlds Yes, all the style changes were automatically added by spotless task. I fixed a few lines where the automatic formatting removed the initial spaces (they were left on purpose to align the content with the previous enumeration, for example). I will leave fixing the formatting for a different PR.

**Comment by jsync-swirlds** [comment]
Not ideal, to be sure. The spotless config definitely needs to be adjusted then (in my opinion); likely very soon before it becomes fixed in stone.

> @jsync-swirlds Yes, all the style changes were automatically added by spotless task. I fixed a few lines where the automatic formatting removed the initial spaces (they were left on purpose to align the content with the previous enumeration, for example). I will leave fixing the formatting for a different PR.

**Review by poulok — APPROVED** [review]

**Review by lpetrovic05 — APPROVED** [review]

**Pull request merged**